docsrc/conf.py

```diff
@@ -7,11 +7,12 @@
 import sys, os
 import sphinx_bootstrap_theme
 
-sys.path.insert(0, os.path.abspath("../../"))
```

Comment:
I'm not sure this is the correct fix. In [`spaghetti`](https://github.com/pysal/spaghetti/blob/d3cbda62ed6cdfd2b6a4fb9bdfcb084610e6031b/docsrc/conf.py#L15), [`giddy`](https://github.com/pysal/giddy/blob/458c06c9346eded33bfa8f3b4de0f40bc467a904/docsrc/conf.py#L23), etc. this path is correct. Maybe we can get @weikang9009 to look at this if she has a moment?

Reply:
Serge helped me find the issue and fixed the problem.

Reply:
It seems that it fixed the problem locally on your machine, but there may be an issue with your directory paths because `sys.path.insert(0, os.path.abspath("../../"))` works for all other PySAL submodules and it works for me locally with `spopt` (aside from the [`calinski_harabasz_score` error](https://github.com/pysal/spopt/issues/79), which is actually an unrelated problem). Best we figure out what's going on with your local build rather than changing this in `main`.